src/caseworker-ref-data/components/caseworker-details/caseworker-details.component.ts

```diff
@@ -0,0 +1,54 @@
+import { CaseWorkerRefDataService } from '../../services/caseworker-ref-data.service';
+import { CaseWorkerRefDataUploadResponse } from '../../models/caseworker-ref-data.model';
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { handleFatalErrors } from '../../utils/caseworker-utils';
+
+@Component({
+  selector: 'app-prd-caseworker-details',
+  templateUrl: './caseworker-details.component.html',
+})
+export class CaseWorkerDetailsComponent {
+
+  public errorDesc: string;
+
+  public constructor(private readonly caseWorkerRefDataService: CaseWorkerRefDataService,
```

Comment:
This looks wrong. That first parameter should also be on a new line. I think lint might complain about this.